src/utils/env.js

```diff
@@ -23,68 +23,70 @@ const homedir = os.homedir();
 const configDirPath = `${homedir}/${configDirName}`;
 const configFilePath = `${configDirPath}/${configFileName}`;
 
-const getReadErrorMessage = (error, path) => {
-	if (error.code === 'EACCES') {
-		return `WARNING: Could not read \`${path}\`: permission denied. Using default config instead.`;
-	}
-	if (error.message.match(/JSON/)) {
-		return `WARNING: Config file at \`${path}\` is not valid JSON. Using default config instead.`;
+const getWriteErrorMessage = path => `WARNING: Could not write to \`${path}\`. Your configuration will not be persisted.`;
```

Comment:
This warning is shown twice in non-interactive mode if set command is used. Two messages are fine but should be more explicit about creating the file.